lib/constants.ts

```diff
@@ -22,3 +22,10 @@ export enum RoutingType {
   CLASSIC = 'CLASSIC',
   DUTCH_LIMIT = 'DUTCH_LIMIT',
 }
+
+export const DEFAULT_POSITIVE_CACHE_ENTRY_TTL = 600; // 10 minutes
+export const DEFAULT_NEGATIVE_CACHE_ENTRY_TTL = 600; // 10 minute
+
+export const ENABLE_PORTION = (portionFlag?: string) => {
+  return portionFlag === 'true';
+};
```

Comment:
nit - can we hardcode `process.env.ENABLE_PORTION` here, so that callers don't have to repeated pass in `portionFlag`?

```suggestion
export const ENABLE_PORTION = process.env.ENABLE_PORTION === 'true';
```